src/get-supported-attributes.ts

```diff
@@ -41,7 +41,7 @@ export function getSupportedAttributes(element: Element | VirtualElement, option
       return roles.application.supported;
     }
     case 'img': {
-      const name = calculateAccessibleName(element, roles.img);
+      const name = calculateAccessibleName(element, roles.image);
```

Comment:
Ah thank you! I actually had this question in the back of my mind which was preferred but kept forgetting to look it up. Ditto for `none` vs `presentation`.

Reply:
Yeah IIRC `none` is now the preferred alias as was introduced to avoid the confusion `presentation` caused - haven’t checked to see what have gone for in this lib…